mediorum/server/serve_blob.go

```diff
@@ -156,82 +156,68 @@ func (ss *MediorumServer) serveBlob(c echo.Context) error {
 		c.SetParamValues(values...)
 	}
 
-	logger := ss.logger.With("cid", cid)
 	key := cidutil.ShardCID(cid)
 
-	// return 403 if the requested CID is delisted
-	shouldCheckDelistStatus := true
-	if checkedDelistStatus, exists := c.Get("checkedDelistStatus").(bool); exists && checkedDelistStatus {
-		shouldCheckDelistStatus = false
-	}
-	if shouldCheckDelistStatus && ss.isCidBlacklisted(ctx, cid) {
-		logger.Info("cid is blacklisted")
-		return c.String(403, "cid is blacklisted by this node")
-	}
-
 	// if the client provided a filename, set it in the header to be auto-populated in download prompt
 	filenameForDownload := c.QueryParam("filename")
 	if filenameForDownload != "" {
 		contentDisposition := mime.QEncoding.Encode("utf-8", filenameForDownload)
 		c.Response().Header().Set("Content-Disposition", fmt.Sprintf(`attachment; filename="%s"`, contentDisposition))
 	}
 
-	// check our bucket and serve the file if we have it
-	if attrs, err := ss.bucket.Attributes(ctx, key); err == nil && attrs != nil {
-		isAudioFile := strings.HasPrefix(attrs.ContentType, "audio")
-		// detect mime type and block mp3 streaming outside of the /tracks/cidstream route
-		if !strings.Contains(c.Path(), "cidstream") && isAudioFile {
-			return c.String(401, "mp3 streaming is blocked. Please use Discovery /v1/tracks/:encodedId/stream")
-		}
+	blob, err := ss.bucket.NewReader(ctx, key, nil)
 
-		blob, err := ss.bucket.NewReader(ctx, key, nil)
-		if err != nil {
-			return err
+	if err != nil {
```

Comment:
Add back a comment here

`// If our bucket does not have the file, find another one`